src/test/java/org/opentripplanner/generate/doc/framework/TemplateUtil.java

```diff
@@ -1,12 +1,29 @@
 package org.opentripplanner.generate.doc.framework;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.util.DefaultIndenter;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.SerializationFeature;
 import org.opentripplanner.standalone.config.framework.json.NodeAdapter;
 
 /**
  * Replace a text in a file wrapped using HTML comments
  */
 public class TemplateUtil {
 
+  public static final ObjectMapper MAPPER = new ObjectMapper();
+  public static final ObjectWriter PRETTY_PRINTER;
```

Comment:
These should be private - they are not constants and can accidentaly be changed form the outside of the class.